includes/polling/os/racomos.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$hardware = trim(snmp_get($device, '.1.3.6.1.2.1.1.5', '-OQv', '', ''), '"');
+$version  = trim(snmp_get($device, '.1.3.6.1.4.1.33555.1.1.2.2', '-OQv', '', ''), '"');
```

Comment:
Can you change these two snmp_gets into multi gets, see `includes/polling/os/netguardian.inc.php` for an example.